README.md

```diff
@@ -354,3 +354,37 @@ The `Table` component requires child components of type `Table.Column`, which pr
 **Usage Example**
 
 [Sample Code](https://github.com/Clever/components/tree/master/docs/TableExample.jsx) ([Live Demo](http://clever.github.io/components/#table))
+
+### Grid
+
+12-column grid component for consistent, simple and flexible layouts.
```

Comment:
It took me a second to parse the hierarchy of `Grid`s. Maybe worth describing the general structure in a sentence or two eg. "A Grid component is made up of any number of `Row` components. Each `Row` component is made of any number of `Col` components."